pipeline/jobs/test-performance.yaml

```diff
@@ -0,0 +1,63 @@
+parameters:
+  - name: displayName
+    type: string
+
+  - name: dependsOn
+    type: object
+    default: ''
+
+  - name: environment
+    type: string
+
+  - name: subscription
+    type: string
+
+  - name: variables
```

Comment:
does this need to be an object?

Reply:
I have no idea why its being done like this, i am just following the methods used by the other tests - i has previously just passed in as a param but was getting issues with the dependencies